File: ios/paystack-ios-3.0.13/Tests/installation_tests/cocoapods/without_frameworks/CocoapodsTest/ViewController.m
//
//  ViewController.m
//  CocoapodsTest
//

#import "ViewController.h"
#import <Paystack/Paystack.h>
#import <Paystack/UIImage+Paystack.h>

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    [Paystack setDefaultPublishableKey:@"test"];
    // Do any additional setup after loading the view, typically from a nib.
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

@end

File: ios/paystack-ios-3.0.13/Tests/installation_tests/cocoapods/with_frameworks/CocoapodsTest/ViewController.swift
//
//  ViewController.swift
//  CocoapodsTest
//

import UIKit
import Paystack

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        Paystack.setDefaultPublishableKey("test")
        // Do any additional setup after loading the view, typically from a nib.
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }


}

